tasks.py

```diff
@@ -309,6 +309,15 @@ def migrate_search(delete=False, index=settings.ELASTIC_INDEX):
     from website.search_migration.migrate import migrate
     migrate(delete, index=index)
 
+@task
+def rebuild_search():
+    """Delete and recreate the index for elasticsearch"""
+    run("curl -XDELETE {uri}/{index}".format(uri=settings.ELASTIC_URI,
```

Comment:
If you want to make sure you delete all versions of the website index, you might want ```curl -XDELETE {uri}/{index}\*```